FPS-Submission_Guidelines.md

```diff
@@ -138,7 +138,7 @@ Upon submission of a PR, a series of technical checks will run on GitHub to veri
 		<li>Each domain must satisfy the /.well-known/ metadata requirement:</li>
 		<ul>
 <li>The /.well-known/ metadata requirement demonstrates that the submitter has administrative access to the domains present in the set, since administrative access is required to modify the /.well-known/ file. This will help prevent unauthorized actors from adding domains to a set. </li>
-<li>The primary domain must serve a JSON file at /.well-known/first-party-set.json. The contents of the file must be identical to the submission. Each member domain must serve a JSON file at /.well-known/first-party-set.json. The contents of the file must name the primary domain. These files must be maintained for the duration of the domain’s inclusion in the set.</li>
+<li>The primary domain must serve a JSON file at /.well-known/first-party-set.json. The contents of the file must be identical to the submission. Each of the associatedSites and serviceSites domain must also serve a JSON file at /.well-known/first-party-set.json. The contents of the first-party-set.json file on these domains must at a minimum name the primary domain. These files must be maintained for the duration of the domain’s inclusion in the set.</li>
```

Comment:
```suggestion
<li>The primary domain must serve a JSON file at /.well-known/first-party-set.json. The contents of the file must be identical to the submission. Each of the set members must also serve a JSON file at /.well-known/first-party-set.json. The contents of the first-party-set.json file on these domains must name the primary domain. These files must be maintained for the duration of the domain’s inclusion in the set.</li>
```

The Submission Guidelines [already define](https://github.com/GoogleChrome/first-party-sets/blob/3035f1f15aea0d34a9ff133c0959d0c0293129ed/FPS-Submission_Guidelines.md#:~:text=A%20set%20member%20is%20a%20domain) "set member", so we can reuse that terminology here instead of enumerating the `associatedSites` and `serviceSites` explicitly. Saying "set member" is more future-proof in case new kinds of subsets exist in the future, and it avoids erroneously excluding the ccTLD domains.

(If you feel so inclined, it would be nice for this usage of "set"member" to link back to the definition. We could do that by adding an anchor to the definition, e.g. using the method described [here](https://stackoverflow.com/a/7015050). Feel free to leave this for a future enhancement, though.)

I would also like to avoid making the implication that anything other than the `"primary"` field is read from the set members' JSON files. We don't verify (or require) anything but the `"primary"` field, so the "at a minimum" phrase feels misleading to me, since it could lead readers to think that any additional fields that they include will also be checked.

Reply:
Thanks for the clarification, makes sense. I can make those changes.

Reply:
(For future reference - you can iterate on a pull request and add more commits to your branch, as it goes through code review, without having to close the PR and open a new PR for each round of feedback. In this particular case, I had written my suggestion as a ["suggested change"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request), so that you could click "commit suggestion" and automatically incorporate the feedback if you agreed with it.)